### ISSUE TITLE
Naming things

### ISSUE THREAD

**Issue by rahul286** [rtCamp/Frappe-Manager#101]
Let's solve #2 problem of Computer Science ([source](https://martinfowler.com/bliki/TwoHardThings.html)) :D

Everywhere we call the thing FM creates is `site` while the things that are within frappe bench are also called `site`

I think what FM creates is actually a bench. Unsure if I can `fm shell <name>` and create more benches within.

**Comment by Xieyt** [comment]
You are correct this is indeed confusing. I will go with your suggestion and refactor code to follow the same.
